### ISSUE TITLE
[BITV]: Replace default background image with darker one to pass a11y contrasts on menu 

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42196]
Right now contrast between foreground and background colors is 4.17:1. This is not enough.

**_Have to be min. 4:5 to 1 in dark theme and light theme._**

![Screenshot from 2023-12-13 09-17-52](https://github.com/nextcloud/server/assets/6078378/40c36cf7-3886-4697-b11f-2389ced14cde)

See: 9.1.4.3/0.1 - Note: For text (including in focused state) under 18 points (or for bold text under 14 points), the contrast ratio between the foreground and background colors should be at least 4.5:1

**Comment by JuliaKirschenheuter** [comment]
already fixed by 
- https://github.com/nextcloud/server/pull/42287 
![Screenshot from 2023-12-18 11-15-00](https://github.com/nextcloud/server/assets/6078378/33bd52c9-1336-49cb-a830-ab4d8e86d6f6)

**Comment by JuliaKirschenheuter** [comment]
not done
